internal/controllers/printresults/print_results.go

```diff
@@ -376,16 +376,8 @@ func (pr *PrintResults) checkIfExistsErrorsInAnalysis() {
 	}
 }
 
-func (pr *PrintResults) printErrors(errorMessage string) {
-	if strings.Contains(errorMessage, messages.MsgErrorPacketJSONNotFound) ||
-		strings.Contains(errorMessage, messages.MsgErrorYarnLockNotFound) ||
-		strings.Contains(errorMessage, messages.MsgErrorGemLockNotFound) ||
-		strings.Contains(errorMessage, messages.MsgErrorNotFoundRequirementsTxt) {
-		logger.LogWarnWithLevel(strings.ReplaceAll(errorMessage, ";", ""))
-		return
-	}
-
-	logger.LogStringAsError(strings.ReplaceAll(errorMessage, ";", ""))
+func (pr *PrintResults) printErrors(err string) {
+	logger.LogStringAsError(strings.ReplaceAll(err, ";", ""))
 }
```

Comment:
I prefer to leave this workaround on print results and not on analyzer. Leaving on print results we just add a new case on this if statement.

Reply:
I don't think I understand the idea, as the error status is set before the print results. How would adding one more case in this if fix the status?

Reply:
I think I confused the correction. Today we log error messages as a warning and end the analysis as an error. This PR removes these error messages to be logged as a warning, and so the analysis ends successfully, correct? I was thinking we would change the log type from warning to error.

Reply:
Previously, if the analysis had any warning, it would end with an error status.

This [workflow](https://github.com/ZupIT/beagle-docs/runs/5422976461) contains an example of this scenario.

Reply:
This status depends on whether the analysis has vulnerabilities or not, so it was necessary to remove warnings from the errors field.

Reply:
Makes sense, I had misunderstood the problem. I don't like the approach used to fix the problem but I can not think anything better, we should find a better approach to implement formatters urgently